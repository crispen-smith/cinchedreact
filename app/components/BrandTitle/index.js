/**
 *
 * BrandTitle
 *
 */

import React from 'react';
// import PropTypes from 'prop-types';
import styled from 'styled-components';

const BT = styled.div`
  flex: 8;
  padding-top: 1.5rem;
  padding-bottom: 0;
  margin: 0;
  line-height: 1.5;
`;

const BrandTitleBase = styled.h1`
  margin: 0;
  font-family: 'Monsieur La Doulaise', cursive;
  font-size: 2.5rem;
`;

const BrandMessage = styled.h2`
  transform: translateX(35px) translateY(-20px);
  font-family: 'Lora', serif;
  font-size: 0.5rem;
  margin: 0;
  padding: 0;
`;

function BrandTitle() {
  return (
    <BT>
      <BrandTitleBase>Cinched Tight</BrandTitleBase>
      <BrandMessage>Celebrate Your Shape</BrandMessage>
    </BT>
  );
}

BrandTitle.propTypes = {};

export default BrandTitle;
