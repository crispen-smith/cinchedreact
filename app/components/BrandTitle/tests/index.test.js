import React from 'react';
import renderer from 'react-test-renderer';

import BrandTitle from '../index';

describe('<BrandTitle />', () => {
  it('Renders a BrandTitle Component', () => {
    const tree = renderer.create(<BrandTitle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
