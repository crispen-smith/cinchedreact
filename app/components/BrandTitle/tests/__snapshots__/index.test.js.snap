// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BrandTitle /> Renders a BrandTitle Component 1`] = `
<div
  className="BrandTitle__BT-iyACyt doctaa"
>
  <h1
    className="BrandTitle__BrandTitleBase-fCqnon fsKsKs"
  >
    Cinched Tight
  </h1>
  <h2
    className="BrandTitle__BrandMessage-bmovll bAcupH"
  >
    Celebrate Your Shape
  </h2>
</div>
`;
